gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskIntegrationSpec.groovy

```diff
@@ -223,9 +223,7 @@ class CreateManifestTaskIntegrationSpec extends GradleIntegrationSpec {
         runTasks(':testCreateManifest')
 
         then:
-        def manifest = CreateManifestTask.jsonMapper.readValue(
-                file('build/deployment/manifest.yml', projectDir).text, Map)
-        manifest.get("extensions").get("product-dependencies").size() == 1
```

Comment:
this is bad style and short-circuits the better introspection that the check on the following line gives you, which is to actually show a diff